checked_yaml/changelog.md

```diff
@@ -1,3 +1,7 @@
+## 1.0.3
```

Comment:
1.0.3-dev

Note we have documented this now :) https://github.com/dart-lang/sdk/wiki/External-Package-Maintenance#making-a-change

Reply:
Ugh...fine